### ISSUE TITLE
Wrong rendering of Unicode characters in ICS invitations

### ISSUE THREAD

**Issue by r-xyz** [usnistgov/NEMO#137]
Hi all,

Thanks as always for such a great tool.
I noticed a minor bug regarding calendar invitation and character encoding.
If an Unicode character (e.g. `®`) is present in `Site title` (under `Customizations-->Application`) or in `RESERVATION_ORGANIZER` (under `settings.py`), the ICS invitation will be rendered incorrectly, showing strange characters.
In some cases (like for `®`), the pair composed by the Unicode character and the one following (e.g. `]`) is swapped for a single Japanese character (e.g. `孫`); in other cases (like for `α`), only the Unicode is swapped for its code  (e.g. `\u03b1`).

I guess the problem is somehow related to https://github.com/usnistgov/NEMO/blob/d9d0fc8c82ab0239fc9d130c4873c83e026525db/NEMO/utilities.py#L446

Googling around, the issue seems related to [missing encoding specification for the ICS file attachment](https://theeventscalendar.com/support/forums/topic/ical-text-encoding/) (which will default to `iso-8859-1` instead of `UTF-8`).

Below you can compare the headers proceeding the ics attachment in base64, taken from an email invitation received from NEMO:
```
Content-Type: text/calendar; method="REQUEST"
MIME-Version: 1.0
Content-Transfer-Encoding: base64
[ICS in base64]
```
or from a correct email invitation from Google Calendar:
```
Content-Type: text/calendar; charset="UTF-8"; method=REQUEST
Content-Transfer-Encoding: quoted-printable
[ ICS in clear text ]
Content-Type: application/ics; name="invite.ics"
Content-Disposition: attachment; filename="invite.ics"
Content-Transfer-Encoding: base64
[ICS in base64]

```
or from a correct email invitation from Thunderbird;
I hope this helps to identify the problem.
Thanks.

**Comment by r-xyz** [comment]
Small update: 
I did some tests adding `charset="UTF-8"` header, but the result did not change. My guess is the problem arises from some conversion happening either in:
https://github.com/usnistgov/NEMO/blob/2952f55a0b05c1a879f402ea0af4ef2925a6aa44/NEMO/views/calendar.py#L1204
or in:
https://github.com/usnistgov/NEMO/blob/d9d0fc8c82ab0239fc9d130c4873c83e026525db/NEMO/utilities.py#L446

**Comment by rptmat57** [comment]
closed by #138
